components/brex_staging/actions/list-transactions-for-selected-cash-account/list-transaction-for-selected-cash-acccount.mjs

```diff
@@ -0,0 +1,21 @@
+import brexApp from "../../brex_staging.app.mjs";
+import common from "../../../brex/actions/list-transaction-for-selected-cash-account/common.mjs";
+
+export default {
+  ...common,
+  name: "List Transactions for Selected Cash Account",
+  description: "Lists all transactions for the specified cash account. [See the docs here](https://developer.brex.com/openapi/transactions_api/#operation/listCashTransactions).",
+  key: "brex_staging-list-transactions-for-selected-cash-account",
```

Comment:
The filename should have the word transactions plural to match the key. `list-transactions-for-selected-cahs-account.mjs`